### PR TITLE
Allow setting the repo file name

### PR DESCRIPTION
**Pull request by sposs**
Use the same switch but use the value as file name. Means it cannot use the default file name ($HOME/.dirac.repo.cfg, will create another fix for that).

### CODE REVIEW

**Comment by sposs** [comment]
Don't merge yet, I need to do something else.

**Comment by sposs** [comment]
Ok, now it's OK

File: Interfaces/scripts/dirac-wms-job-submit.py [hunk 1]
```diff
@@ -30,10 +30,13 @@
 from DIRAC.Interfaces.API.Dirac                              import Dirac
 unprocessed_switches = Script.getUnprocessedSwitches()
 use_repo = False
+repo_name = ""
 for sw, value in unprocessed_switches:
   if sw.lower() in ["r", "usejobrepo"]:
     use_repo = True
-dirac = Dirac(use_repo)
+    repo_name = value
+    repo_name = repo_name.replace(".cfg", ".repo")
```
Comment (chaen) [inline Interfaces/scripts/dirac-wms-job-submit.py]: ".repo" ? the other change you did are with ".rep". Is it normal ?

Comment (sposs) [inline Interfaces/scripts/dirac-wms-job-submit.py]: It doesn't really matter. The only relevant thing was to change from .cfg to anything else. This particular line above is just to make sure no one uses .cfg as extension.

**Pull request merged**